docs/experience/index.md

```diff
@@ -0,0 +1,6 @@
+title: Overview
```

Comment:
Does this mean that the page title will be just "Overview"?  If so, wouldn't we want something like "Sematext Experience Overview"?

Reply:
_Sematext Experience Overview_ would be great, and if so, then you can remove the `<h3>` _Overview_ right underneath.

Reply:
This is just a placeholder for now, I'll send another PR for Overview and Getting Started pages.